### PR TITLE
[Tyr] Indexing data for Elasticsearch2 en Elasticsearch7 while doing Import_data depending on configuration.

### PR DESCRIPTION
**Pull request by azime**
Jira: [NAVP-1952](https://jira.kisio.org/browse/NAVP-1952)

### CODE REVIEW

File: source/tyr/tyr/tasks.py [hunk 1]
```diff
@@ -208,30 +208,35 @@ def send_to_mimir(instance, filename, family_type):
     job.instance = instance
     job.state = 'running'
 
-    dataset = models.DataSet()
-    dataset.family_type = 'mimir'
-    dataset.type = 'fusio'
+    if is_activate_autocomplete_version(7):
```
Comment (kadhikari) [inline source/tyr/tyr/tasks.py:211]: We can add a comment above this line 'We create a dataset for each autocomplete version if active'

File: source/tyr/tests/integration/mimir_test.py [hunk 2]
```diff
@@ -71,7 +62,21 @@ def test_mimir_family_type_poi(create_instance, enable_mimir):
         assert len(actions) == 2  # poi2mimir + finish
 
 
-def test_mimir_ntfs_false(create_instance, enable_mimir):
+def test_mimir2_and_mimir_family_type_poi(create_instance, enable_mimir2_and_mimir):
+    with app.app_context():
+        instance = models.Instance.query.get(create_instance)
+        actions = []
+        actions.extend(tasks.send_to_mimir(instance, 'test.poi', 'poi'))
+        assert actions[0].task == 'tyr.binarisation.poi2mimir'
```
Comment (kadhikari) [inline source/tyr/tests/integration/mimir_test.py:70]: assert actions[1].task ??

**Review by kadhikari — COMMENTED** [review]
great job :+1: 
Seems fine to me. Since we can't test all with existing unit and integration test, I propose to test with a tyr local for each functionality.

File: source/tyr/tests/integration/datapost_test.py [hunk 3]
```diff
@@ -62,7 +62,7 @@ def get_jobs_from_db(id=None):
         return models.Job.get(id=id)
 
 
-def test_post_pbf(create_instance_fr):
+def test_post_pbf(create_instance_fr, enable_mimir2):
```
Comment (benoit-bst) [inline source/tyr/tests/integration/datapost_test.py:65]: i don't know if it is easy, but maybe `enable_mimir_es7` is better ?

**Review by benoit-bst — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

**Review by kadhikari — APPROVED** [review]

**Pull request merged**